### PR TITLE
Add FAQ document

### PR DESCRIPTION
**Pull request by akning-ms**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: documentation/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+If you are new to Swagger/OpenAPI, you can refer to this [document](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/80/Getting-started-with-OpenAPI-specifications)
+
+If you are new to Swagger PR review process, you can refer to this [flowchart](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/212/Swagger-PR-Review)
+
+You can refer to this [document](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/49/Request-Access-to-Azure-SDK-repos) to get read permission to submit PR.
+
+## How to fix validation failure?<a name="validation"></a>
+| Validation | Description | How to fix |
+| --- | --- | --- |
+| [Modal Validation](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md) | Validates models defined for body parameters and responses, and matches them against the expected input and output of an operation. | [Here](https://aka.ms/ci-fix#model-validation) |
```
Comment (lirenhe) [inline documentation/FAQ.md:21]: typo: Model

Comment (lirenhe) [inline documentation/FAQ.md:21]: For the description of model validation, consider reference https://github.com/Azure/oav/blob/master/README.md#what-does-the-tool-do-what-issues-does-the-tool-catch

Comment (akning-ms) [reply documentation/FAQ.md]: Thanks, fixed

Comment (akning-ms) [reply documentation/FAQ.md]: Thanks, changed.

File: documentation/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+This page is intended to answer questions frequently asked during Azure Swagger PR review.
+
+1. [I am new to Swagger/OpenAPI, How should I start?](#Onboard)
+2. [How to fix validation failure?](#validation)
+3. [How to generate SDK from Swagger?](#SDK)
+4. [How to generate document](#doc)
+5. [If need further help, who can we contact?](#contact)
+
+## How to onboard PR review Process?<a name="Onboard"></a>
+If you are new to Swagger/OpenAPI, you can refer to this [document](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/80/Getting-started-with-OpenAPI-specifications)
```
Comment (josefree) [inline documentation/FAQ.md:12]: Wonder if this document is better? https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/71/AME-onboard-home

File: documentation/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+| [Avocado](https://github.com/Azure/avocado/blob/master/README.md) | Avocado validates folder structure and configuration.  | [Here](https://aka.ms/ci-fix#avocado) |
+| [Semantic](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md) | Semantic validation enforces correctness on the swagger specific elements. Such as paths and operations. Ensure the element definition meet the OpenApi 2.0 specification. | [Here](https://aka.ms/ci-fix#semantic-validation) |
+| Prettier  | Code formatter for Swagger | [Here](https://aka.ms/ci-fix#prettier-check) |
+| BranchProtectionForPrivateRepo | BranchProtectionForPrivateRepo is designed to always fail to prevent from merging PR into master which is not allow in private swagger repo as it will cause issue to sync up from public repo.  | You can ignore this failure |
+
+Refer to [Document](ci-fix) for how to run these tools locally
```
Comment (josefree) [inline documentation/FAQ.md:29]: this Document link is not valid

File: documentation/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+## How to generate SDK from Swagger?<a name="SDK"></a>
+SDK generation is triggered as soon as your OpenAPI specification (a.k.a swagger) PR is created/committed/merged in to the azure-rest-api-specs repository.
```
Comment (josefree) [inline documentation/FAQ.md:32]: If you are working in the public repository, SDK generation is triggered ....
If you are working in the private repository, please refer to this document to manually generate SDKs of different languages, including Python, Ruby, Go, C#, etc.

**Review by josefree — COMMENTED** [review]
better to clarify we only have SDK auto-generation support in public repository.  else just fine.

File: documentation/FAQ.md (hunk 3 above)
Comment (akning-ms) [reply documentation/FAQ.md]: fixed, thanks

File: documentation/FAQ.md (hunk 4 above)
Comment (akning-ms) [reply documentation/FAQ.md]: good suggestion, changed

**Pull request merged**